src/index.html

```diff
@@ -1,13 +1,375 @@
+                <img
+                  src="./images/product/photo-product-06.png"
+                  alt="photo-product-06"
+                  class="card__photo"
+                >
+                <div class="card__info">
+                  <div class="card__subtitle page__subtitle">Cookies</div>
+                  <div class="card__text page__text">
+                    Oatmeal cookies with golden raisins, orange zest and chocolate chips
+                  </div>
+                </div>
+                <a href="#Contact_us" class="card__link link">Order</a>
+              </article>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- #endregion -->
+      <!-- #region about-us -->
+      <section class="about-us" id="About-us">
+        <div class="container">
+          <div class="about-us__content">
+            <div class="about-us__top">
+              <h2 class="about-us__top__title page__title">
+                Few words About us
+              </h2>
+              <p class="about-us__top__text page__text">
+                Like many design teams, Onfido made the transition from Abstract + Sketch,
+                to Figma in the last couple of years. One of the common resistance points internally before making that switch,
+                was Figma&#39;s lack of Git-style branching that Abstract enabled.
+                The design team had come to rely on branching and had built a lot of our processes around it.
+              </p>
+            </div>
+            <div class="about-us__bottom">
+              <div class="about-us__bottom__numbers">
+                <div class="about-us__bottom__number">2015</div>
+                <div class="about-us__bottom__subtitle">Foundation year</div>
+              </div>
+
+              <div class="about-us__bottom__numbers">
+                <div class="about-us__bottom__number">23</div>
+                <div class="about-us__bottom__subtitle">Stores</div>
+              </div>
+
+              <div class="about-us__bottom__numbers">
+                <div class="about-us__bottom__number">155</div>
+                <div class="about-us__bottom__subtitle">Companies</div>
+              </div>
+
+              <div class="about-us__bottom__numbers">
+                <div class="about-us__bottom__number">10k+</div>
+                <div class="about-us__bottom__subtitle">Clients</div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- #endregion -->
+    </main>
+    <!-- #region footer -->
+    <footer class="footer">
+      <div class="container">
+        <div class="footer__content">
+          <div class="footer__menu">
+            <div class="footer__menu__wrapper">
+              <a href="#" class="footer__menu__logo-link">
+                <img
+                  class="footer__menu__logo"
+                  src="./images/logo-footer.svg"
+                  alt="Bakerlab logo">
```

Comment:
Check and fix it everywhere
```suggestion
                <img
                  class="footer__menu__logo"
                  src="./images/logo-footer.svg"
                  alt="Bakerlab logo"
                >
```